canvas_modules/common-canvas/src/command-actions/deleteObjectsAction.js

```diff
@@ -172,16 +172,29 @@ export default class DeleteObjectsAction extends Action {
 	}
 
 	createActionLabel() {
+
+		const stringList = [
+			{ label: "Nodes", val: this.nodesToDelete.length + this.supernodesToDelete.length },
+			{ label: "Comments", val: this.commentsToDelete.length },
+			{ label: "Links", val: this.linksToDelete.length }
+		];
+		const dynamicStringKey = "action.delete" + stringList
+			.filter((v) => v.val).map((o) => o.label)
+			.join("");
+
 		if (this.areDetachableLinksSupported) {
-			return this.labelUtil.getActionLabel(this, "action.deleteNodesCommentsLinks",
+			return this.labelUtil.getActionLabel(this, "action.delete" + dynamicStringKey,
 				{ nodes_count: this.nodesToDelete.length + this.supernodesToDelete.length,
 					comments_count: this.commentsToDelete.length,
 					links_count: this.linksToDelete.length
 				});
 		}
-		return this.labelUtil.getActionLabel(this, "action.deleteNodesComments", {
-			nodes_count: this.nodesToDelete.length + this.supernodesToDelete.length,
-			comments_count: this.commentsToDelete.length
-		});
+
+		return this.labelUtil.getActionLabel(this, dynamicStringKey,
+			{
+				nodes_count: this.nodesToDelete.length + this.supernodesToDelete.length,
+				comments_count: this.commentsToDelete.length,
+				links_count: this.linksToDelete.length
+			});
 	}
 }
```

Comment:
Have you thought about a regression test for this?

Reply:
Yes,I'm working on them!!